mbed_greentea/mbed_target_info.py

```diff
@@ -167,15 +163,17 @@ def get_yotta_target_from_local_config(yotta_json='.yotta.json'):
       }
     }
     """
-    result = None
-    if os.path.exists(yotta_json):
-        try:
-            gt_logger.gt_log("parsing local file '%s' for target information"% yotta_json)
-            with open(yotta_json, 'r') as f:
-                result = parse_yotta_json_for_build_name(json.load(f))
-        except Exception as e:
-            print(str(e))
-    return result
+    if not os.path.exists(yotta_json):
+        return None
+
+    try:
+        gt_logger.gt_log("parsing local file '%s' for target information"% yotta_json)
+
+        with open(yotta_json, 'r') as f:
+            return parse_yotta_json_for_build_name(json.load(f))
+    except Exception as e:
```

Comment:
Please refine this except to catch only the exceptions that you intend to generate. As it stands this will eat Control-c and other spurious exceptions.